src/modules/SX126x.h

```diff
@@ -678,6 +678,13 @@ class SX126x: public PhysicalLayer {
     */
     float getSNR();
 
+    /*!
+      \brief Set DIO2 to function as RF switch (default in Semtech example designs).
+
+      \returns \ref status_codes 
+    */
+    int16_t setDio2AsRfSwitch(bool enable = false);
```

Comment:
Change default enable value to true. If user is calling this method, it is probably because he wants to enable the RF control.

Reply:
Right.